devices/ubisys.js

```diff
@@ -750,6 +843,12 @@ module.exports = [
             return {'default': 1, 'meter': 3};
         },
         onEvent: async (type, data, device) => {
+            if (data.type === 'attributeReport' && data.cluster === 'seMetering') {
```

Comment:
You can probably change this into a function, from a quick glance can grab the correct endpoint from the data object so no need to hardcode 3 and 5

Reply:
yes that would be a possibility. Unfortunately, I'm not that good at writing functions. and the endpoint is already hardcoded everywhere on all devices anyway.
```
        endpoint: (device) => {
            return {'default': 1, 's1': 2, 's2': 3, 'meter': 4};
```